http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BaseRuntimeInterpreter.scala

```diff
@@ -97,8 +97,8 @@ abstract private[util] class BaseRuntimeInterpreter[F[_]: Async: ContextShift: L
       val labelFound = config.welderConfig.updateWelderLabel.exists(runtime.labels.contains)
 
       val imageChanged = runtime.runtimeImages.find(_.imageType == Welder) match {
-        case Some(welderImage) if welderImage.imageUrl != config.imageConfig.welderImage => true
-        case _                                                                           => false
+        case Some(welderImage) if welderImage.imageUrl != config.imageConfig.welderGcrImage => true
```

Comment:
If welderImage is from dockerhub, it'll never equal to `welderGceImage`, and we shouldnt' try to update the image...

Reply:
Right now, the label on which we update welder is `saturnVersion` - until we change that we won't be supporting updating welder from Dockerhub anyways. AoU also didn't seem to care much about having welder auto-updated

Reply:
I realize that's the assumption here...but once aou start using the new flag, welder image URL will be something like "broadinstitute/welder-server:7ae1a62", so this condition will always be true, but we should have it as false cuz we don't want to update it in this case

Reply:
It's the same as it is now though. We won't be updating the image either way because both the label needs to be there too (see line 104)